nvflare/private/fed/client/client_engine.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, client, client_name, sender, args, rank, workers=5):
         self.args = args
         self.rank = rank
         self.client.process = None
-        self.client_executor = ProcessExecutor(client.client_name)
+        self.client_executor = ProcessExecutor(client.client_name, os.path.join(args.workspace, "startup"))
```

Comment:
Should we pass args.workspace to ProcessExecutor? It should have access to the whole workspace.